test/unit/spec/MfaVerify_spec.js

```diff
@@ -3172,6 +3175,59 @@ function (Okta,
       });
     });
 
+    Expect.describe('Supports chains of MFA_REQUIRED responses', function () {
+      itp('Mfa verify follows password re-auth without trapping the auth response', function () {
+        return setupPassword().then(function (test) {
+          spyOn(RouterUtil, 'handleResponseStatus').and.callThrough();
+          $.ajax.calls.reset();
+          test.form.setPassword('Abcd1234');
+          test.form.setRememberDevice(true);
+          test.setNextResponse(resAllFactors);
+          test.form.submit();
+          return Expect.waitForVerifyQuestion(test);
+        })
+        .then(function (test) {
+          test.questionForm = new MfaVerifyForm($sandbox.find('.o-form'));
+          expect(test.questionForm.isSecurityQuestion()).toBe(true);
+          // trapAuthResponse does not prevent handleResponseStatus from being called
+          expect(RouterUtil.handleResponseStatus.calls.count()).toBe(1);
+        });
+      });
+      itp('Any factor change through the dropdown traps auth response', function () {
+        return setupSMS().then(function (test) {
+          spyOn(RouterUtil, 'handleResponseStatus').and.callThrough();
+          test.setNextResponse(resChallengeSms);
+          test.form.smsSendCode().click();
+          return tick(test);
+        })
+        .then(function (test) {
+          test.setNextResponse(resAllFactors);
+          test.beacon.dropDownButton().click();
+          clickFactorInDropdown(test, 'QUESTION');
+          return Expect.waitForVerifyQuestion(test);
+        })
+        .then(function () {
+          // trapAuthResponse prevents handleResponseStatus from being called
+          expect(RouterUtil.handleResponseStatus.calls.count()).toBe(0);
+        });
+      });
+      itp('Verify inline totp with factor Okta Verify Push traps auth response', function () {
+        return setupOktaPush().then(function (test) {
+          return setupPolling(test, resRejectedPush)
+          .then(function () {
+            spyOn(RouterUtil, 'handleResponseStatus').and.callThrough();
+            test.totpForm = new MfaVerifyForm($($sandbox.find('.o-form')[1]));
+            test.totpForm.inlineTOTPAdd().click();
```

Comment:
clicking this link does not make any call or anything right? It only shows the textbox for the user to manually input the code, so not sure this test is needed.

Reply:
Changed this test to call verify